students/kaczmarek_katarzyna/lesson_04_unit_testing/fibonacci_numbers.py

```diff
@@ -0,0 +1,34 @@
+# Fibonacci: 1 1 2 3 5 8 13 ...
+def fib(n):
+    if isinstance(n, bool):
```

Comment:
mozna by tutaj dodac również sprawdzenie czy n jest None: `if n is None or isnsta...`, żeby nie potrzebnie nie uruchamiać dalszej części funkcji na nie poprawnych typach/wartościach.